test/enrollment_test.rb

```diff
@@ -58,4 +58,16 @@ def test_sanitizes_participation
 
     assert_equal expected, sanitized
   end
+
+  def test_gets_average_participation_for_all_years
+    e = Enrollment.new({:name => "ACADEMY 20",
+                        :kindergarten_participation => {
+                          2010 => 0.3915, # .392
+                          2011 => 0.35356,# .354
+                          2012 => 0.2677  # .268
+    }})
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.<br>Space inside } missing.